_includes/build/cancelling_jobs.html

```diff
@@ -0,0 +1,20 @@
+<h3>Step 1: Obtain an Access Token</h3>
+	<p>
+		See <a href="#authorization" class="in-text__link">Authentication and Authorization</a> above.
+	</p>
+<h3>Step 2: Send a Request to Delete a Job</h3>
```

Comment:
Can we change Delete to Cancel?

Reply:
Done! Corrected a few typos that were in there as well